acra-censor/handlers/handlers_util.go

```diff
@@ -151,6 +151,9 @@ func handleSelectWherePattern(queryNodes, patternNodes []sqlparser.SQLNode) bool
 	patternWhereDetected := false
 	queryWhereDetected := false
 	for index, patternNode := range patternNodes {
+		if index >= len(queryNodes) {
```

Comment:
maybe it makes sense to compare slice's lengths before loop?
```
if len(queryNodes) != len(patternNodes)
```

Reply:
Yes, actually we can perform this comparison before loop. But instead  `!=` it's better `<`, because length of query is allowed to be equal or greater than pattern. The intention here was to prevent case when pattern is greater than query, and index out of range when use `queryNodes[index]`